equipment.lua

```diff
@@ -21,7 +21,358 @@ local function give_player_fighting_equipment(player)
 end
 
 local function give_player_tools(player)
-	if (player.force.technologies["steel-processing"].researched) then
+	if (player.force.technologies["steel-processing"].researched) then--Grief detection
+--Written by Mylon, 2017-2018
```

Comment:
Suddenly an antigrief module appeared.

Reply:
I placed the wrong code on the `pulll request, but didnt know how to cancel the pull request of this file.
placed the correct one on another pull request

Reply:
You can push the correct file to your patch-1 branch which will update this pull request.